openedx/features/edly/context_processor.py

```diff
@@ -1,12 +1,13 @@
 from math import floor
 
+from django.conf import settings
 from django.core.cache import cache
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 
 CACHE_NAME = 'context_processor.dynamic_theming'
 EDLY_CACHE_NAME = 'context_processor.edly_app'
-DEFAULT_SERVICES_COOKIE_EXPIRY = 180
-CACHE_TIMEOUT = 60
+DEFAULT_SERVICES_NOTIFICATIONS_COOKIE_EXPIRY = 180  # value in seconds
+CACHE_TIMEOUT = 60  # value in seconds
```

Comment:
We need to set this value to 15 minutes.

Reply:
Feedback Implemented. `CACHE_TIMEOUT` is now configurable from site configurations

Reply:
Feedback Implemented